README.md

```diff
@@ -1,7 +1,17 @@
 # Lean blueprints
 
 This is a [plasTeX](https://github.com/plastex/plastex/) plugin allowing
-to write blueprints for Lean projects.
+to write blueprints for Lean 4 projects.
 
 Needs graphviz and its deve libraries.
 `sudo apt install graphviz libgraphviz-dev`
+
+## Migration from Lean 3
+
+The plugin has dropped support for Lean 3. If you have a Lean 3 blueprint project and want to use the latest version of the plugin, you need to do the following:
+
+1. Port your Lean 3 code to Lean 4, likely with the help of [mathport](https://github.com/leanprover-community/mathport)
+2. Set up [doc-gen4](https://github.com/leanprover/doc-gen4) for your project, publish it somewhere, e.g. https://your-docgen4-url-base/
+3. The new plugin jumps to the documentation generated by `doc-gen4` for `\lean{YourLean4Theorem}` if you have specified `\dochome{https://your-docgen4-url-base/}` in your `web.tex` file. This is equivalent to the old plugin, which jumped to the source code of the theorem in your GitHub repository for `\lean{your_lean3_theorem}`. You need to carefully port all `\lean{your_lean3_theorem}` to `\lean{YourLean4Theorem}`, this could be just a search and replace with the CamelCase version of your theorem name, or it could have been renamed in your Lean 4 porting process.
```

Comment:
I just realized that this line written in early commits is inaccurate, definitions would have case porting issues but not theorems according to Mathlib naming conventions, the example here is totally wrong (but not misleading because no one would be actually misled I guess). Will fix it in a follow-up PR (but I might wait for some other issues) or feel free to just fix it.